sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsMeta.scala

```diff
@@ -154,6 +162,15 @@ abstract class RapidsMeta[INPUT <: BASE, BASE, OUTPUT <: BASE](
    */
   final def canThisBeReplaced: Boolean = cannotBeReplacedReasons.exists(_.isEmpty)
 
+  /**
+   * Returns the list of reasons the entire plan can't be replaced. An empty
+   * set means the entire plan is ok to be replaced, do the normal checking
+   * per exec and children.
+   */
+  final def canAnyOfPlanBeReplacedReasons: Seq[String] = {
```

Comment:
`canAnyOfPlan` makes me think these are reasons part or all of the plan can be replaced which is the opposite of what it means.  Should this be named something like `entirePlanExcludedReasons`?

Reply:
sounds better to me